Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.UIElements.cs

```diff
@@ -83,36 +83,43 @@ public void SetWidth(string name, StyleLength value)
 
         private async void SetDisplay(VisualElement element, bool value, bool fade)
         {
-            if (element.style.opacity == 0 && !value)
+            try
```

Comment:
I don't think Task.Delay can ever throw with correct arguments, so revert

Reply:
Sure, but there could be null refs from other things in here. Leaving as is.

Reply:
What? No it can't. If this ever throws it's a bug we should fix. I don't want nonsensse try/catch around everything.

Reply:
the visual element could have been destroyed, you don't have control over the lifetime of objects. So a NRE is possible to be thrown in this block.

Reply:
Also ***how would you ever know if the exceptions fall through cracks bc you don't have this change***?

Reply:
What do you mean? Unity logs uncaught exceptions

Reply:
Not for async void, which is the purpose of the change

Reply:
reverted